web_programming/fetch_github_info.py

```diff
@@ -8,19 +9,23 @@
 for more information around suggested workarounds and removal dates.
 """
 
+"""
+Created by lawric1 on 24/11/20
+
+To generate your personal access token visit https://github.com/settings/tokens
+"""
 
-import requests
 
 _GITHUB_API = "https://api.github.com/user"
 
 
-def fetch_github_info(auth_user: str, auth_pass: str) -> dict:
+def fetch_github_info(auth_user: str, auth_token: str) -> dict:
     """
     Fetch GitHub info of a user using the requests module
     """
-    return requests.get(_GITHUB_API, auth=(auth_user, auth_pass)).json()
+    return requests.get(_GITHUB_API, auth=(auth_user, auth_token)).json()
```

Comment:
Authorization tokens are not passed in the `auth` tuple, but they are put in the request headers. Please read the documentation properly.